twoliter/src/lock.rs

```diff
@@ -38,6 +38,12 @@ pub(crate) struct LockedImage {
     pub(crate) manifest: Vec<u8>,
 }
 
+impl PartialEq for LockedImage {
+    fn eq(&self, other: &Self) -> bool {
+        self.source == other.source && self.digest == other.digest
+    }
+}
```

Comment:
I think the image digest should be sufficient for equivalency, no?

Reply:
The source could change with vendor but the digest could theoretically be the same